_posts/0000-01-02-BNEE-dev.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![92982240-crispy-chunky-chips-in-a-metal-dish-on-a-wooden-table-within-the-citadel-victoria-rabat-gozo-malta-e](https://user-images.githubusercontent.com/65422311/82094441-faac3d80-96f4-11ea-9825-b4db77fe4449.jpg)